### ISSUE TITLE
Crash on camera/screen share start when targeting Android 11

### ISSUE THREAD

**Issue by cswagner** [LogmeinRescue/Android-SDK#48]
Testing on a device running Android 11 with `compileSdkVersion` and `targetSdkVersion` set to `30`, I'm getting a crash when starting either a camera or screen share session:

```
java.lang.SecurityException: getDataNetworkTypeForSubscriber
        at android.os.Parcel.createExceptionOrNull(Parcel.java:2373)
        at android.os.Parcel.createException(Parcel.java:2357)
        at android.os.Parcel.readException(Parcel.java:2340)
        at android.os.Parcel.readException(Parcel.java:2282)
        at com.android.internal.telephony.ITelephony$Stub$Proxy.getNetworkTypeForSubscriber(ITelephony.java:8762)
        at android.telephony.TelephonyManager.getNetworkType(TelephonyManager.java:3021)
        at android.telephony.TelephonyManager.getNetworkType(TelephonyManager.java:2985)
        at com.logmein.rescuesdk.internal.jb$a.onSignalStrengthsChanged(SourceFile:53)
        at android.telephony.PhoneStateListener$IPhoneStateListenerStub.lambda$onSignalStrengthsChanged$18(PhoneStateListener.java:1222)
        at android.telephony.-$$Lambda$PhoneStateListener$IPhoneStateListenerStub$lP7_Xy6P82nXGbUQ_ZUY6rZR4bI.run(Unknown Source:4)
        at android.os.Handler.handleCallback(Handler.java:938)
        at android.os.Handler.dispatchMessage(Handler.java:99)
        at android.os.Looper.loop(Looper.java:223)
        at android.app.ActivityThread.main(ActivityThread.java:7656)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:592)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:947)
```

The failing method requires the `READ_PHONE_STATE` permission which I don't want to request from the user. Could the case where this permission is denied be handled gracefully to prevent the crash?

**Comment by tamasgyongyosi** [comment]
Hi @cswagner ,

This is because of using DeviceInfoExtension. You can prevent the SecurityException If you are not using this extension.
Also you have to enable "Classic display for mobile" setting in Admin Center. You can find this option in the "Organization" tab when you select a Technician Group in the left side panel.
![ac_classic_display](https://user-images.githubusercontent.com/4340880/115853904-f8d11b00-a429-11eb-98b2-f22ee97dadb4.PNG)

Furtheremore you have to select "Launch Display Viewing Session" during the session creation process in the Technician Console to get thorough to screen sharing.
![tc_classic_display](https://user-images.githubusercontent.com/4340880/115853918-fbcc0b80-a429-11eb-9469-06518eff9a33.PNG)

Please let me know if this solution works for you.

**Comment by tamasgyongyosi** [comment]
Hi @cswagner ,

We have just released a new version of the SDK (3.7.2). If you use the DeviceInfoExtension then the SDK acquire the READ_PHONE_STATE permission from the user. If the user deny this permission then there is no SecurityException and the Technician Console does not get network type information from the phone.

**Comment by tamasgyongyosi** [comment]
I'm glad to hear that :) Thank you for bringing this issue to our attention.
